renderlib/graphics/GestureGraphicsGL.cpp

```diff
@@ -413,16 +413,8 @@ bool
 GestureRendererGL::pick(SelectionBuffer& selection,
                         const Gesture::Input& input,
                         const SceneView::Viewport& viewport,
-                        Gesture::Graphics& graphics)
+                        uint32_t& selectionCode)
 {
-  // If we are in mid-gesture, then we can continue to use the retained selection code.
-  // if we never had anything to draw into the pick buffer, then we didn't pick anything.
-  // This is a slight oversimplification because it checks for any single-button release
-  // or drag: two-button drag gestures are not handled well.
-  if (input.clickEnded() || input.isDragging()) {
-    return graphics.m_retainedSelectionCode != SelectionBuffer::k_noSelectionCode;
-  }
-
```

Comment:
this piece of logic is lifted out of here and put in ViewerWindow just before `pick` is called.